src/main/java/org/black_ixx/bossshop/pointsystem/BSPointsPluginVotingPlugin.java

```diff
@@ -9,12 +10,12 @@ public class BSPointsPluginVotingPlugin extends BSPointsPlugin {
     public BSPointsPluginVotingPlugin() {
         super("VotingPlugin", "VP");
     }
-
+    VotingPluginMain votingPlugin = (VotingPluginMain) Bukkit.getPluginManager().getPlugin("VotingPlugin");
```

Comment:
I'm not sure how exactly the code is set up for _when_ this would actually run. Is it possible this might cause problems if the VotingPlugin is not enabled yet when this class is hit?

Reply:
Hmm.  I don't know enough to know... Would you prefer I put the variable as a local variable in each method rather than as an instance variable? 

The constructor right above that does this:
`    public BSPointsPlugin(String main_name, String... nicknames) {
        this.name = main_name;
        this.nicknames = nicknames;
        this.available = Bukkit.getPluginManager().getPlugin(name) != null;
    }`
    And all the methods were using a VotingPlugin class before.
    
    I have loaded and run this several times and it works.  That said, I don't know enough about plugin loading to make any guarantees.  (Sorry!)

Reply:
Did you try it with and without the voting plugin? If it eureka without the voting plugin present, I'm cool with it.

Reply:
It now works with and without votingplugin present.